components/chat_widget/src/components/ocs-chat/ocs-chat.css

```diff
@@ -193,6 +212,26 @@
   --code-text-assistant-color: var(--message-assistant-text-color);
   --code-border-assistant-color: color-mix(in srgb, var(--message-assistant-bg-color) 90%, black 10%);;
 
+  --confirmation-overlay-bg-color: rgba(0, 0, 0, 0.5);
+  --confirmation-dialog-bg-color: var(--chat-window-bg-color);
+  --confirmation-dialog-border-color: var(--chat-window-border-color);
+  --confirmation-dialog-shadow-color: var(--chat-window-shadow-color);
+  --confirmation-dialog-border-radius: 0.75em;
+  --confirmation-title-color: var(--input-text-color);
```

Comment:
Rather just default this to a fixed value since it's not clear that it should be linked to the `input text color`